docs/source/azure/gettingstarted.rst

```diff
@@ -11,25 +11,43 @@ Getting Started
 Install Cloud Custodian and Azure Plugin
 ----------------------------------------
 
-The Azure provider must be installed as a separate package in addition to c7n.
+Cloud Custodian is a Python application and supports Python 2 and 3 on Linux and Windows.
```

Comment:
MacOS?

Reply:
Well we don't actually run any tests on MacOS/FreeBSD :)